src/components/InventoryGroups/utils/api.js

```diff
@@ -0,0 +1,21 @@
+import { instance } from '@redhat-cloud-services/frontend-components-utilities/interceptors/interceptors';
+import { INVENTORY_API_BASE } from '../../../api';
+
+//had to replace the instance.post with axios.post otherwise I had an error because chrome is missing
```

Comment:
The comment doesn't seem to be appropriate here (you use instance.post)

Reply:
Removed it